aeon/io/video.py

```diff
@@ -14,7 +14,7 @@ def frames(data):
     filename = None
     index = 0
     try:
-        for frameidx, path in zip(data._frame, data._path):
+        for frameidx, path in zip(data._frame, data._path, strict=False):
```

Comment:
`zip` does have a `strict` argument since version 3.10 as defined in [PEP 618](https://peps.python.org/pep-0618/). It was introduced explicitly as a way to avoid the issues with `zip_longest` where `None` might be introduced multiple times as keys.